operator/api/v1alpha1/manifest_types.go

```diff
@@ -96,20 +97,25 @@ const (
 
 // ManifestSpec defines the specification of Manifest
 type ManifestSpec struct {
```

Comment:
Im missing a field to pass the CR data as unstructured here, is that somewhere else?